lib/util/ios-proxy.js

```diff
@@ -14,6 +14,10 @@ const ROOT_URI = getPref("extensions." + id + ".sdk.rootURI");
 const TOOLS_URI = ROOT_URI + "tools/";
 
 let proxyProcess;
+let binaryName = "ios_webkit_debug_proxy";
+if (Runtime.OS == "WINNT") {
+  binaryName = "ios-webkit-debug-proxy.exe";
```

Comment:
The change from underscores to dashes is kind of random... Maybe we should rename the exe just for uniformity?